### ISSUE TITLE
White screen on iOS 9.3

### ISSUE THREAD

**Issue by Almaju** [VeliovGroup/Meteor-Files#450]
After investigation about my website showing a white page on some devices (and especially on iOS 9.3 devices), I discovered ostrio:files was causing it.

> [Error] SyntaxError: Unexpected keyword 'const'. Const declarations are not supported in strict mode.
> 	(fonction anonyme) (ostrio_files.js:4419)

Here is the corresponding code:
```
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//                                                                                                                     //
// node_modules/meteor/ostrio_files/node_modules/fs-extra/lib/index.js                                                 //
//                                                                                                                     //
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
                                                                                                                       //
'use strict'                                                                                                           // 1
                                                                                                                       // 2
const assign = require('./util/assign')                                                                                // 3
                                                                                                                       // 4
const fs = {}                                                                                                          // 5
                                                                                                                       // 6
// Export graceful-fs:                                                                                                 // 7
assign(fs, require('./fs'))                                                                                            // 8
// Export extra methods:                                                                                               // 9
assign(fs, require('./copy'))                                                                                          // 10
assign(fs, require('./copy-sync'))                                                                                     // 11
assign(fs, require('./mkdirs'))                                                                                        // 12
assign(fs, require('./remove'))                                                                                        // 13
assign(fs, require('./json'))                                                                                          // 14
assign(fs, require('./move'))                                                                                          // 15
assign(fs, require('./move-sync'))                                                                                     // 16
assign(fs, require('./empty'))                                                                                         // 17
assign(fs, require('./ensure'))                                                                                        // 18
assign(fs, require('./output'))                                                                                        // 19
assign(fs, require('./path-exists')) 
```

**Comment by dr-dimitru** [comment]
Duplicate, see #445 #444

**Comment by dr-dimitru** [comment]
Hello @Almaju ,

This should be fixed in [`v1.8.0`](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.8.0) please update to the latest release and let me know if it solves this issue.

**Comment by dr-dimitru** [comment]
Closed due to silence at issue owner end.
Feel free to reopen it in case if the issue still persists on your end.
